integration/dockerfiles/Dockerfile_test_cache

```diff
@@ -16,7 +16,14 @@
 # If the image is built twice, /date should be the same in both images
 # if the cache is implemented correctly
 
-FROM gcr.io/google-appengine/debian9@sha256:1d6a9a6d106bd795098f60f4abb7083626354fa6735e81743c7f8cfca11259f0
-RUN date > /date
-COPY context/foo /foo
-RUN echo hey
+FROM alpine as base_stage
```

Comment:
Would this Dockerfile fail integration tests if caching didn't work? As in, if kaniko built it a second time without the cache during the tests, I'm not sure if container-diff would catch the difference (that's why I had `RUN date > /date` in the old version, since container-diff would fail if the `date` layer wasn't pulled from the cache).

Reply:
Great point! I changed this to reproduce the caching issue I was hitting, but dropped the date thing. I'll double check.